src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/13.diff.sql

```diff
@@ -421,4 +421,32 @@ AS
 	from dbo.TaskInfo task INNER JOIN @availableJobs availableJob ON task.TaskId = availableJob.TaskId
 
     COMMIT TRANSACTION
-GO
\ No newline at end of file
+GO
+
+/*************************************************************
+    Stored procedures for get disabled indexes
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     GetDisabledIndexess
+--
+-- DESCRIPTION
+--     Get disabled indexes
+--
+CREATE OR ALTER PROCEDURE [dbo].[GetDisabledIndexess]
```

Comment:
We should consider to put it into enable and disable index stored procedure. That would also help improve reliability on disable/enable.